templates/apps/sumo-aws-apps/guardduty-org/guardduty-org-configuration.yaml

```diff
@@ -294,14 +311,14 @@ Resources:
         - 'CAPABILITY_AUTO_EXPAND'
       StackInstancesGroup:
         - Regions:
-            -  !Select [0, !Split [",", !Ref EnabledRegions]]
+            - !Ref 'AWS::Region'
```

Comment:
Do we need to deploy this stackset only in one region ? not in all guardduty enabled regions

Reply:
Role it deploy only one region